### ISSUE TITLE
Add GPU usage

### ISSUE THREAD

**Issue by kyleman** [josephsl/resourceMonitor#14]
It would be really nice to see how much processing is done by the GPU of your machine. 

I think the keyboard shortcut for this information should be NVDA+shift+4. The battery information this offers, is already provided with the NVDA+shift+B command.

**Comment by josephsl** [comment]
Hi, interesting. Do you have use cases? Note that in Windows 11, Task Manager can display GPU usage information. Also note: I’m no longer maintaining this add-on. Thanks.
